### ISSUE TITLE
Support --log-cli

### ISSUE THREAD

**Issue by jiasli** [pytest-dev/pytest#6373]
In the current pytest 5.3.2, `--log-cli` can only be configured via ini. 

https://github.com/pytest-dev/pytest/blob/b532d15fe7543b7ee895cbbc101fcd0fe2543312/src/_pytest/logging.py#L209-L214

It will be great to support `--log-cli` as a command line option.

**Comment by RonnyPfannschmidt** [comment]
The option override option is suppose to be used for that

**Comment by thisch** [comment]
Alternatively, you can also use `--log-cli-level=LEVEL`.

**Comment by nicoddemus** [comment]
@jiasli this is intentional, because the `-o` allows you to change any ini option through the command line:

```
pytest -o log_cli=true
```

The original logging plugin contained ini option alternatives for every command-line option, and when it was introduced into pytest's core we kept all the original options/command-line arguments for backward compatibility.

Currently we try to avoid having duplicated command-line and configuration options, because the `-o` switch solves this nicely. The `--log-cli` was added after the logging plugin introduction to the core, so it follows this new approach.
